api/v08/kogitoserverlessworkflow_types.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package v08
 
 import (
+	"encoding/xml"
```

Comment:
@davidesalerno shouldn't we be dropping these from here and use the sdk-go instead?

Reply:
before a full review, @davidesalerno let's align it.

Reply:
@davidesalerno @spolti Anything I can do to help?